WBooks/Library/LibraryView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LibraryView.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 12/08/21.
+//
+
+import Foundation
+import UIKit
+
+final class LibraryView: NibView {
+    
+    @IBOutlet weak var tableView: UITableView!{
+        didSet{
+            let libraryTableViewCell = LibraryTableViewCell()
+            let nib = UINib.init(nibName: libraryTableViewCell.nibName, bundle: nil)
```

Comment:
the .init is unnecesary, you can do:

`let nib = UINib(nibName: libraryTableViewCell.nibName, bundle: nil)`

Reply:
Done